models/CourseModel.php

```diff
@@ -29,6 +29,8 @@ class CourseModel {
 	const STATUS_DRAFT      = 'draft';
 	const STATUS_AUTO_DRAFT = 'auto-draft';
 	const STATUS_PENDING    = 'pending';
+	const STATUS_PRIVATE    = 'private';
+	const STATUS_SCHEDULE   = 'future';
```

Comment:
I think the const name should have had`STATUS_FUTURE` since WP is calling it future.